<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KalkuM</class>
 <widget class="QMainWindow" name="KalkuM">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KalkuM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonClose">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>650</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Zamknij</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>30</y>
      <width>670</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>2x2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3x3</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>150</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Wybierz rozmiar macierzy</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Dodawanie</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Odejmowanie</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Mno≈ºenie</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>322</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>Odwrotna</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Wyznacznik</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridWidget_3" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>108</width>
      <height>84</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <property name="spacing">
      <number>30</number>
     </property>
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
       </property>
       <property name="maxLength">
        <number>9999</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
       </property>
       <property name="maxLength">
        <number>9999</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
       </property>
       <property name="maxLength">
        <number>9999</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
       </property>
       <property name="maxLength">
        <number>999</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridWidget_4" native="true">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>108</width>
      <height>84</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <property name="spacing">
      <number>30</number>
     </property>
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
       </property>
       <property name="maxLength">
        <number>999</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
       </property>
       <property name="maxLength">
        <number>9999</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
       </property>
       <property name="maxLength">
        <number>9999</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="lineEdit_8">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
       </property>
       <property name="maxLength">
        <number>9999</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Macierz A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Macierz B</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridWidget_5" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>380</y>
      <width>108</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <property name="spacing">
      <number>30</number>
     </property>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_9">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
       </property>
       <property name="maxLength">
        <number>4</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_11">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
       </property>
       <property name="maxLength">
        <number>4</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_12">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
       </property>
       <property name="maxLength">
        <number>4</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEdit_10">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
       </property>
       <property name="maxLength">
        <number>4</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>350</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>WYNIK</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>490</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Wyznacznik:</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>171</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>113</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_17"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_18"/>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit_13"/>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit_16"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_14"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit_21"/>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="lineEdit_19"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEdit_15"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_20"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>120</y>
      <width>171</width>
      <height>121</height>
     </rect>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>113</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_26"/>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit_25"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_27"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_23"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEdit_24"/>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit_22"/>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="lineEdit_28"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_29"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit_30"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>380</y>
      <width>171</width>
      <height>121</height>
     </rect>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>113</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_31">
        <property name="maxLength">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit_32">
        <property name="maxLength">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_33">
        <property name="maxLength">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_34">
        <property name="maxLength">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEdit_35">
        <property name="maxLength">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit_36">
        <property name="maxLength">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="lineEdit_37">
        <property name="maxLength">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_38">
        <property name="maxLength">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit_39">
        <property name="maxLength">
         <number>4</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_skalar" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>490</y>
      <width>120</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QLabel" name="wynik_skalar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_skalar2" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>490</y>
      <width>120</width>
      <height>21</height>
     </rect>
    </property>
    <widget class="QLabel" name="wynik_skalar2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <zorder>pushButtonClose</zorder>
   <zorder>comboBox</zorder>
   <zorder>label</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>gridLayoutWidget_2</zorder>
   <zorder>gridWidget_4</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>gridWidget_5</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>widget</zorder>
   <zorder>widget_2</zorder>
   <zorder>widget_3</zorder>
   <zorder>gridWidget_3</zorder>
   <zorder>widget_skalar</zorder>
   <zorder>widget_skalar2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionZapisz"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionZapisz">
   <property name="text">
    <string>Zapisz</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>KalkuM</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>657</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>738</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
